docs/guides/func_yaml.md

```diff
@@ -87,12 +87,6 @@ The Kubernetes namespace where your function will be deployed.
 
 The language runtime for your function. For example `python`.
 
-### `trigger`
```

Comment:
This should be replaced with an entry for `template`.
```
### `template`
The source code template tailored for the invocation event that triggers
your function. Possible values are `http` for plain HTTP requests, and `event`
CloudEvent triggered functions.
```

Reply:
Thanks very much for pointing out the obvious omission; a casualty of merge it appears.